packages/theme/src/theme/_variables.scss

```diff
@@ -167,6 +167,11 @@ $btn-secondary-color: #fff !default;
 $btn-secondary-bg: $brand-secondary !default;
 $btn-secondary-border: darken($btn-secondary-bg, 5%) !default;
 
+$btn-tertiary-info-color: $st-tropaz !default;
+$btn-tertiary-default-color: $dark-silver !default;
```

Comment:
You can remove those 2 variables. The colors are not defined specifically for tertiary but for all buttons

Reply:
updated